demo/demo.component.html

```diff
@@ -75,11 +75,19 @@ <h2>Column Definition Example</h2>
     textAlign="center"
     width="100px"
     sorting="false"
-    filtering="false">
+    type="template"
+    [filtering]="false">
     <ng-template let-data="data">
       <button (click)="onItemClick($event, data)">Edit</button>
       <button (click)="onItemClick($event, data)">Delete</button>
     </ng-template>
+    <ng-template context="header">
+      <div style="text-align: center">
```

Comment:
no inline, pls move this here https://github.com/raketasoft/ng2-grid/blob/master/demo/styles.css